### PR TITLE
Awall sentry

### PR DESCRIPTION
**Pull request by awall1010**
Added sentry as singer connector and added requirements

### CODE REVIEW

File: singer-connectors/tap-fastly/requirements.txt [hunk 1]
```diff
@@ -1,2 +1 @@
--e git://github.com/splitio/tap-fastly.git@master#egg=tap-fastly
-
+-e git://github.com/splitio/tap-fastly.git@awall-tap-fastly#egg=tap-fastly
```
Comment (pmasse) [inline singer-connectors/tap-fastly/requirements.txt]: I don't think this should go in.

File: pipelinewise/cli/pipelinewise.py [hunk 2]
```diff
@@ -137,7 +137,7 @@ def create_filtered_tap_properties(self, target_type, tap_type, tap_properties,
                     self.logger.error('Stream %s has no metadata with no breadcrumbs: %s.', tap_stream_id, metadata)
                     raise Exception(f'Missing metadata in stream {tap_stream_id}')
 
-                selected = table_meta.get('selected', False)
+                selected = table_meta.get('selected', False) #gets something from metadata to decide if sync or not
```
Comment (pmasse) [inline pipelinewise/cli/pipelinewise.py]: I don't think this should go in

File: pipelinewise/cli/pipelinewise.py [hunk 3]
```diff
@@ -937,7 +937,7 @@ def run_tap(self):
             tap_properties_fastsync,
             fastsync_stream_ids,
             tap_properties_singer,
-            singer_stream_ids
+            singer_stream_ids
```
Comment (pmasse) [inline pipelinewise/cli/pipelinewise.py]: I don't think this should go in

File: install.sh [hunk 4]
```diff
@@ -166,14 +166,15 @@ DEFAULT_CONNECTORS=(
     tap-fastly
     tap-jira
     tap-kafka
-    #tap-mysql
+    tap-mysql
```
Comment (pmasse) [inline install.sh]: You should not uncomment that. It caused issues with the build

File: install.sh [hunk 5]
```diff
@@ -166,14 +166,15 @@ DEFAULT_CONNECTORS=(
...
     tap-pagerduty
     tap-postgres
     tap-s3-csv
     tap-salesforce
     tap-snowflake
     tap-zendesk
-    #target-s3-csv
```
Comment (pmasse) [inline install.sh:176]: Just add tap-sentry but don't uncomment the other. It caused some issues with the build

**Review by pmasse — CHANGES_REQUESTED** [review]

**Review by awall1010 — COMMENTED** [review]
re commented out connectors and fixed fastly requirements

**Review by pmasse — APPROVED** [review]

**Pull request merged**